src/m365/planner/commands/task/task-add.ts

```diff
@@ -339,6 +362,19 @@ class PlannerTaskAddCommand extends GraphCommand {
       return 'Specify either assignedToUserIds or assignedToUserNames but not both';
     }
 
+    if (args.options.priority !== undefined) {
+      if (typeof args.options.priority === "number") {
+        // Number validation
+        if (isNaN(args.options.priority) || args.options.priority < 0 || args.options.priority > 10) {
+          return 'priority should be a number between 0 and 10.';
+        }
```

Comment:
I guess we allow not only integers 🤔
I think this gives and error response like `Cannot convert the literal '*.*' to the expected type 'Edm.Int32'.`
do you think we could upgrade this validation a bit 🤔? or maybe also validate from 0-10 based on list of allowed values  [0,1,2,... 10]

Reply:
Good point. Right now `1.5` is indeed considered as a valid number.
Maybe I can just add an extra validation `... && Number.isInteger(args.options.priority)`. Or use indeed the array of valid values. Not sure what is the clearest/cleanest. ☺️